chia/wallet/puzzles/dao_proposal.clsp

```diff
@@ -183,7 +148,7 @@
       my_amount
     )
     (if coin_id_list
-        (assert (> (f vote_amount_list) 0)
+        (if (> (f vote_amount_list) 0)
```

Comment:
Why this change?  I think this is what assert does anyways yeah?

Reply:
leaving it as if was a mistake, heldover from debugging fixed now